administration/src/application/components/ApplyController.tsx

```diff
@@ -57,14 +66,25 @@ const ApplyController = () => {
         application: applicationInput,
       },
     })
+
+    setFormSubmitted(true)
+    setState(() => ApplicationForm.initialState)
```

Comment:
This should only be called, if `addBlueEakApplication` is successful. we need to await the promise, check if the status code is 200 and only then do setFormSubmitted(true), ....
Also: Please set the loading prop of ApplicationForm.Component according to the loading state of the request.

Reply:
@f1sh1918 Sorry,  I think I didn't explain it well: Please await the promise returned by `addBlueEakApplication()`, and add some try/catch block. In the success case, `setFormSubmitted(true)` and `setState(()=> ApplicationForm.initialState)` should be called. In the error case, a toast should be displayed.

Reply:
Ah okay that makes sense. Maybe I also didn't Listen carefully. Gonna adjust it

Reply:
Yeah I think so!